spire-server.yaml

```diff
@@ -47,6 +47,9 @@ pipeline:
       echo ${goversion} > .go-version
       cat .go-version
 
+      go get github.com/sigstore/cosign/v2@v2.2.1
```

Comment:
If possible, could you leave a comment with the CVE ID: CVE-2023-46737

Reply:
Added